### ISSUE TITLE
Remove the need to manually register

### ISSUE THREAD

**Issue by renz45** [optick/codemirror-mode-elixir#1]
Hi there, thanks a bunch for putting this together. I was wondering, would you mind auto instantiating this mode with the CodeMirror instance similar to how the other modes work? Here is an example:

https://github.com/codemirror/CodeMirror/blob/master/mode/css/css.js#L10

The idea is, that all you need to do is include the mode after CodeMirror itself and it auto registers itself with CodeMirror. I'm doing this right now with the following small changes to your code:

```javascript
(function (global, factory) {
  typeof exports === 'object' && typeof module !== 'undefined' ? module.exports = factory()(CodeMirror) :
  typeof define === 'function' && define.amd ? define(factory) :
  (global.registerElixirMode = factory()(CodeMirror));
}(this, (function () { 'use strict';
...
```

I'm guessing this code was originally generated by rollup. I'm thinking it might need a commonjs rollup config alongside the umd to accomplish this? 

Thanks again :)

**Comment by ianwalter** [comment]
Yup, I've been meaning to look into this. Ill do this sometime within the next few days, thanks.

**Comment by ianwalter** [comment]
Should be fixed in v1.1.0

**Comment by renz45** [comment]
Thanks!
